src/core_ct/slice.py

```diff
@@ -32,15 +32,15 @@ def __init__(self, data: np.ndarray, pixel_dimensions: tuple[float, float]):
         self.data: np.ndarray = data
         self.pixel_dimensions: tuple[float, float] = pixel_dimensions
 
-    def trim(self, axis: int, loc_start: int, loc_end: int | None = None) -> Slice:
+    def trim(self, axis: int, left_index: int, right_index: int | None = None) -> Slice:
         """
         Create new slice by trimming off a specified amount on the requested axis.
 
         Remove unwanted space around a core that must be removed before analysis.
         A new `Slice` object will be created by taking a slice of the data array
-        between indices `start_loc` and `len(axis) - end_loc`.
-        By default this function is symmetric so the same amount will be taken from
-        either ends of the specified axis. The new slice has the same pixel dimensions.
+        between indices `left_index` and `right_index`. By default this function
+        is symmetric so the same amount will be taken from either ends of the 
```

Comment:
Maybe we should specify that the amount it trims symmetrically depends on `left_index`